packages/vapor/scss/typekit.scss

```diff
@@ -385,3 +395,10 @@ p,
     @extend %book;
     @extend %subdued;
 }
+
+code,
```

Comment:
I wouldn't modify the `code` element, otherwise it's harder to use the styleguide in other websites

Reply:
done deal